src/main/kotlin/no/nav/syfo/client/pdl/PdlClient.kt

```diff
@@ -81,32 +88,71 @@ class PdlClient(
     }
 
     private suspend fun getFnr(ident: String): HttpResponse? {
-        val token = azureAdClient.getSystemToken(urls.pdlScope)?.accessToken
-        val bearerTokenString = "Bearer $token"
         val graphQuery = this::class.java.getResource(IDENTER_QUERY)?.readText()?.replace("[\n\r]", "")
             ?: throw FileNotFoundException("Could not found resource: $IDENTER_QUERY")
         val requestBody = PdlRequest(graphQuery, Variables(ident))
-        return try {
-            client.post(urls.pdlUrl) {
-                headers {
-                    append(PDL_BEHANDLINGSNUMMER_HEADER, BEHANDLINGSNUMMER_DIGITAL_OPPFOLGINGSPLAN)
-                    append(HttpHeaders.ContentType, ContentType.Application.Json)
-                    append(HttpHeaders.Authorization, bearerTokenString)
+        return postCallToPdl(requestBody)
+    }
+
+    private suspend fun getPerson(ident: String): HttpResponse? {
+        val graphQuery = this::class.java.getResource(PERSON_QUERY)?.readText()?.replace("[\n\r]", "")
+            ?: throw FileNotFoundException("Could not found resource: $PERSON_QUERY")
```

Comment:
Hm, vil FileNotFoundException være litt misvisende? Kanskje bedre med IllegalArgumentException eller egen/custom exception?

Reply:
..samme med de andre filenotfound

Reply:
Det virker ikke misvisende for meg, men jeg tenker at den kan kanskje bli `IOException`?

Reply:
Høres bra ut :)